homeassistant/components/sensor/mercedesme.py

```diff
@@ -0,0 +1,127 @@
+"""
+Support for Mercedes cars with Mercedes ME.
+"""
+import logging
+import datetime
+
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = ['mercedesme']
+
+DOMAIN = 'mercedesme'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the sensor platform."""
+    if discovery_info is None:
+        return
+
+    controller = hass.data[DOMAIN]['controller']
+
+    if not controller.cars:
+        return False
+
+    for car in controller.cars:
+        add_devices([Sensor('fuelLevelPercent', car, controller)])
+        add_devices([Sensor('fuelRangeKm', car, controller)])
+        add_devices([Sensor('serviceIntervalDays', car, controller)])
+        add_devices([Sensor('odometerKm', car, controller)])
+        add_devices([Sensor('doorsClosed', car, controller)])
+        add_devices([Sensor('windowsClosed', car, controller)])
+        add_devices([Sensor('locked', car, controller)])
+        add_devices([Sensor('tireWarningLight', car, controller)])
```

Comment:
Those for states look like binary-only. I guess that it would make sense to move them to a `binary_sensor` platform.